homeassistant/components/http.py

```diff
@@ -185,6 +204,51 @@ def register_path(self, method, url, callback, require_auth=True):
         """ Regitsters a path wit the server. """
         self.paths.append((method, url, callback, require_auth))
 
+    @Throttle(MIN_SEC_SESSION_CLEARING)
```

Comment:
I would like to suggest that we break out Session support to a separate class which gets instantiated by the server.